amishasao.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "@amishasao"
+project: "RoboSense"
+---
+
+# Project Name: RoboSense
+
+## Summary
+
+What are you going to build? What does it do?
+I am going to be building a soft robotic glove for individuals with limited hand motor function to improve functionality faster and be able to grip, hold, and release objects with ease. This could be a potentially helpful tool for stroke and other patients attempting to gain back their motor function via therapy or rehabilitation. The glove itself is going to do two main functions: sense pressure (from the fingertips and wrist) and rotate the appendages (based on the pressure reading). So, for example, if I wanted to pick up a book from my desk, I would get to the book, put pressure on my wrist (which will rotate the glove and, thus, my hand downward), put pressure on my fingertips to try to hold the book, hold the book (given the glove's grip in place), and realease the pressure from my fingertips when wanting to drop the book.
+
+Why are you excited to build it?
+I am excited to build this because I believe that it can have real life implications and become a cost-effective option for patients to regain their hand motor function. Additionally, I am really excited to 3d print nylon and experiment with haptic sensors judging the movement of the wearable glove.
+
+## Plan
+
```

Comment:
I see you've got quite a strong plan! Can you please include any lnks of guides you'd be following or any inspiration you've had? thanks (its fine if there's none, just let me know!)

Reply:
My main inspiration was that I have a skinny wrist, which makes it kinda hard to do things like handstands--so I was intrigued by motor function at first. I then looked into it and found something like my project idea that was being developed by Harvard scientists to improve hand motor function in rehabilitation patients. The core function seemed similar to: https://www.instructables.com/DIY-Robotic-Hand-Controlled-by-a-Glove-and-Arduino/, so I decided to give it a try:)

Here are a couple more links:
- https://youtu.be/cxZJSMguo_g
- https://www.instructables.com/Arduino-Flex-Sensor-Glove/